config/defaults.ini

```diff
@@ -54,53 +54,49 @@ manifestdir = ${tmpdir}/manifest
 mounts_add =
 
 # Packages to install in the buildroot
-packages_add =
-  attr
-  awscli
-  ca-certificates
-  coreutils
-  cpio
-  curl
-  dbus-user-session
-  dpkg-dev
-  endless-ca-cert
-  eos-keyring
-  eos-tech-support
-  fdisk
-  flatpak
-  gir1.2-flatpak-1.0
-  gir1.2-glib-2.0
-  gir1.2-ostree-1.0
-  git
-  gnupg
-  json-glib-tools
-  jq
-  libfsapfs-utils
-  mount
-  openssh-client
-  ostree
-  p7zip-full
-  parted
-  pigz
-  python3
-  python3-apt
-  python3-dev
-  python3-gi
-  python3-requests
-  python3-venv
-  qemu-utils
-  rsync
-  udev
-  util-linux
-  unzip
-  wget
-  xz-utils
-
-# Apt archive cache directory
-aptcache_dir = ${build:contentdir}/archives
-
-# Apt archive cache maximum size in bytes (default 2 GB)
-aptcache_max_size = 2147483648
+# NOTE: In eos7, the "buildroot" doesn't have a package manager.
+# This list is preserved as a reference while we work on eos7.
```

Comment:
Please just delete the lists -- as a reference we can look at eos6 branch or git history

Reply:
Done in 38c0c375df6550c5d7f95bd95ca272d91b2061da